backends/imgui_impl_vulkan.cpp

```diff
@@ -969,11 +970,39 @@ bool ImGui_ImplVulkan_CreateDeviceObjects()
         check_vk_result(err);
     }
 
+    if (v->DescriptorPool == VK_NULL_HANDLE)
+    {
+        // This is taken from the examples, but probably way oversized.
```

Comment:
Not only this is oversized, it contains all kinds of resources never to be used by dear-imgui.

Reply:
As far as I could tell IMGUI only ever allocates a single descriptor of type VK_DESCRIPTOR_TYPE_COMBINED_IMAGE_SAMPLER but I don't know enough about imgui to judge that. But thats pretty much exactly my point: The maintainers of the vulkan backend will know a lot better which descriptors are needed than any value a user of the library could ever come up with.